cypress/e2e/HomePage/list.project.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Test homepage: project list
+
+  Scenario: Open existing project
```

Comment:
In this file, you should just check if the list of project is corresponding to the local storage.
AND
This test should be in another file named `open.existing.project.feature` to fit the other files name pattern.

Reply:
no file anymore... where is the test to `check if the list of project is corresponding to the local storage` ?